data_collection/web/guides.rst

```diff
@@ -713,3 +713,34 @@ before `trackPageView` was triggered.
 
 We call this procedure a "manual conversion". Manual conversion doesn't cause an additional conversion event to be tracked like the automatic conversion does.
 Automatic conversion tracking requires a "source" event that is analyzed and if it fits some goal definition then it causes an addition conversion event.
+
+Anonymous tracking
+------------------
+
+You can set JavaScript Tracking Client to mark requests to be anonymized. This feature can be useful when you want to use a consent manager on your website and collect full data only from those users who gave consent to be tracked.
+
+To set JavaScript Tracking client to mark requests as anonymized call :ref:`setUserIsAnonymous<jtc-api-setUserIsAnonymous>`
+
+.. code-block:: javascript
+
+    _paq.push(["setUserIsAnonymous"]);
+
+From now on all following requests send by :ref:`trackPageView<jtc-api-trackPageView>` or any other function that sends requests to CPP, will be marked as a requst that should be anonymized. :ref:`Learn more how Piwik PRO anonymizes users data<https://piwik.pro/blog/how-to-do-useful-analytics-without-personal-data/>`.
```

Comment:
CPP shortcut is not used earlier in guides:
```suggestion
From now on all following requests send by :ref:`trackPageView<jtc-api-trackPageView>` or any other function that sends requests to Collecting and Processing Pipeline (CPP), will be marked as a requst that should be anonymized. :ref:`Learn more how Piwik PRO anonymizes users data<https://piwik.pro/blog/how-to-do-useful-analytics-without-personal-data/>`.
```

Reply:
changed